arm-sql/2014-04-01/swagger/serverDisasterRecoveryConfiguration.json

```diff
@@ -0,0 +1,400 @@
+            "post": {
+                "tags": [
+                    "ServerDisasterRecoveryConfiguration"
+                ],
+                "operationId": "Servers_ForceFailoverAllowDataLossDisasterRecoveryConfiguration",
+                "description": "Begins failover for the Azure SQL server disaster recovery configuration with the given name.",
+                "parameters": [
+                    {
+                        "name": "serverDisasterRecoveryConfigurationName",
+                        "in": "path",
+                        "required": true,
+                        "type": "string",
+                        "description": "The name of the Azure SQL server disaster recovery configuration to start failover."
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ServerNameParameterGeneric"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": ""
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "ServerDisasterRecoveryConfiguration": {
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/ServerDisasterRecoveryConfigurationProperties",
+                    "description": "Gets or sets the properties representing the Server Disaster Recovery Configuration."
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/SubResource"
+                }
+            ],
+            "description": "Represents a server disaster recovery configuration."
+        },
+        "ServerDisasterRecoveryConfigurationList": {
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ServerDisasterRecoveryConfiguration"
+                    },
+                    "description": "Gets or sets the list of Azure Sql Server disaster recovery configurations."
+                }
+            },
+            "description": "Represents the response to a List Azure Sql Server disaster recovery configuration request."
+        },
+        "ServerDisasterRecoveryConfigurationProperties": {
+            "properties": {
+                "autoFailover": {
+                    "type": "string",
+                    "description": "Whether or not automatic failover is enabled."
+                },
+                "failoverPolicy": {
```

Comment:
Need to confirm with team what the valid values are.

---
In reply to: [80168118](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80168118) [](ancestors = 80168118)